tools/drsnoop.py

```diff
@@ -48,22 +52,52 @@
 parser.add_argument("-n", "--name",
                     type=ArgString,
                     help="only print process names containing this name")
+parser.add_argument("-v", "--verbose", action="store_true",
```

Comment:
could you add this flag description in man page?

Reply:
Oh, sorry, I forgot this.